package.json

```diff
@@ -2,7 +2,9 @@
     "name": "@fremtind/jkl",
     "private": true,
     "scripts": {
-        "clean": "lerna run --parallel clean",
+        "clean": "run-p clean:*",
+        "clean: root": "rimraf node_modules/",
+        "clean: packages": "lerna run --parallel clean",
```

Comment:
```suggestion
        "clean:root": "rimraf node_modules/",
        "clean:packages": "lerna run --parallel clean",
```

Reply:
Veldig fan av suggestion og multiline suggestions og add suggestion to batch!

Reply:
Bra catch! Og ja, det er sykt nice! Jeg glemmer å bruke add to batch, men er sykt smooth å kunne ta inn endringer med ett trykk.